tests/app/service/test_rest.py

```diff
@@ -456,3 +456,127 @@ def test_default_permissions_are_added_for_user_service(notify_api,
             service_permissions = json_resp['data']['permissions'][str(sample_service.id)]
             from app.dao.permissions_dao import default_service_permissions
             assert sorted(default_service_permissions) == sorted(service_permissions)
+
+
+def test_add_existing_user_to_another_service(notify_api, notify_db, notify_db_session, sample_service):
+    with notify_api.test_request_context():
+        with notify_api.test_client() as client:
+
+            # check which users part of service
+            user_already_in_service = sample_service.users[0]
+            auth_header = create_authorization_header(
+                path='/service/{}/users'.format(sample_service.id),
+                method='GET'
+            )
+
+            resp = client.get(
+                '/service/{}/users'.format(sample_service.id),
+                headers=[('Content-Type', 'application/json'), auth_header]
+            )
+
+            assert resp.status_code == 200
+            result = json.loads(resp.get_data(as_text=True))
+            assert len(result['data']) == 1
+            assert result['data'][0]['email_address'] == user_already_in_service.email_address
+
```

Comment:
Verify only one user on service. Call post to add, verify 2 users on service, the original one and the one added.